inc/spotim-widgets.php

```diff
@@ -47,7 +47,9 @@ public function __construct() {
     public function widget( $args, $instance ) {
 
         $options = SpotIM_Options::get_instance();
-        $spot_id = $options->get( 'spot_id' );
+
+        // ignoring warning as variable is used in included template.
+        $spot_id = $options->get( 'spot_id' ); //phpcs:ignore

```

Comment:
Can you also add which phpcs rule you're ignoring?
Like ```// phpcs:ignore WordPress.Security.EscapeOutput```

Reply:
It was giving warning for `Unused variable $spot_id`, since the variable is used in included file later on so ignored, couldn't find a specific flag for unused variable.

Reply:
Did you try running PHPCS with `-p -s -v` attributes? It'll show specific tag for the error.

Reply:
Cool, got it.